src/components/tl/ucp/tl_ucp_lib.c

```diff
@@ -22,10 +22,15 @@ UCC_CLASS_INIT_FUNC(ucc_tl_ucp_lib_t, const ucc_base_lib_params_t *params,
     UCC_CLASS_CALL_SUPER_INIT(ucc_tl_lib_t, &ucc_tl_ucp.super,
                               &tl_ucp_config->super);
     memcpy(&self->cfg, tl_ucp_config, sizeof(*tl_ucp_config));
+    status = ucc_mrange_uint_copy(&self->cfg.allreduce_sra_kn_radix,
+                                  &tl_ucp_config->allreduce_sra_kn_radix);
```

Comment:
perhaps add copy also for allreduce_kn_radix (not sra) so that can be used for complete allreduce tuning